### PR TITLE
splitting up some docs, closes #4155

### LINKED ISSUE #4155: Split config docs from 04-schema into 07-config or such
That way we could do a proper section per config setting, and get deep linking to precise settings for free.

### PR DESCRIPTION
**Pull request by alcohol**
Work in progress, feedback please.

I feel like it doesn't really make sense to put config **after** community. However, because of the numbering of the filenames and this actually being reflected in the urls generated, this PR might break some external links. Should I revert that change and move config to the last page (07)?

### CODE REVIEW

File: doc/03-cli.md [hunk 1]
```diff
@@ -424,7 +426,7 @@ php composer.phar config repositories.foo vcs https://github.com/foo/bar
 ## create-project
 
 You can use Composer to create new projects from an existing package. This is
-the equivalent of doing a git clone/svn checkout followed by a composer install
+the equivalent of doing a git clone/svn checkout followed by a "composer install"
```
Comment (nevvermind) [inline doc/03-cli.md]: Ticks instead of quotes?

Comment (alcohol) [inline doc/03-cli.md]: I'm merely trying to follow the styling used in general. I found that often "" are used when the subject doesn't directly relate to the paragraph or an inline code block might distract from the flow of reading.

/cc @Seldaek got any feedback on desired use of quotes or backticks here?

File: doc/06-config.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+# Config
+
+This chapter will describe the `config` section of the `composer.json` 
+[schema](04-schema.md).
+
+## config <span>([root-only](04-schema.md#root-package))</span>
+
+A set of configuration options. It is only used for projects.
+
+## Properties
```
Comment (Seldaek) [inline doc/06-config.md]: I think I'd remove lines 6 to 10 above and use `##` for the properties so they're linkable.

Comment (Seldaek) [inline doc/06-config.md]: Well they might be linkable anyway but still, unnecessary nesting IMO

Comment (alcohol) [inline doc/06-config.md]: Roger

**Pull request merged**